### PR TITLE
[contributing] Add CI topic for new contributors

### PR DESCRIPTION
**Pull request by jverkoey**

### CODE REVIEW

**Comment by jverkoey** [comment]
This PR affects our team culture docs and requires approval from everyone.

File: contributing/README.md
Comment (romoore) [inline contributing/README.md:37]: I would like to see CI run on all new PRs. I think it's valuable for contributors to get automated build/clang-format feedback before a team member looks at changes. It will also save us (those with write access) from having to manually comment on broken/unformatted code.

File: contributing/README.md
Comment (romoore) [inline contributing/README.md]: I assume this is [`core-ios-team`](https://github.com/orgs/material-components/teams/core-ios-team) and not [`mdc-ios`](https://github.com/orgs/material-components/teams/mdc-ios)? Could you please link to the explicit team so readers know we're using these terms as proper nouns and not general concepts?

File: contributing/README.md
Comment (romoore) [inline contributing/README.md]: Rewording/suggestion: "by a member of (whatever membership is required)". I'm not sure what it means to be a member of the repository.

**Review by romoore — DISMISSED** [review]

File: contributing/README.md
Comment (jverkoey) [reply contributing/README.md:37]: Please file a feature request with kokoro for this.

File: contributing/README.md
Comment (jverkoey) [reply contributing/README.md]: I updated the language here. Please resolve accordingly once you've re-reviewed.

**Review by codeman7 — APPROVED** [review]

File: contributing/README.md
Comment (romoore) [reply contributing/README.md:37]: I can't find "kokoro" under the Google organization on GitHub nor in a google search. Can you please provide a link to their bug tracker?

Comment (jverkoey) [reply contributing/README.md:37]: [go/kokoro-bug](http://go/kokoro-bug)

File: contributing/README.md
Comment (romoore) [reply contributing/README.md]: What's the reason for excluding any future subteams of [mdc-ios](https://github.com/orgs/material-components/teams/mdc-ios) from automatic CI kick-offs?

Comment (jverkoey) [reply contributing/README.md]: I don't believe this content as-written precludes the possibility of us expanding the scope to include other teams in the future. As we stand right now, these are the two teams that define who has write access to the repo.

Comment (romoore) [reply contributing/README.md]: Interesting. Then should we change the wording to just reflect "users with write access" instead of naming teams? That seems more future-proof.  Sorry for my initial suggestion, but I thought we were only granting kokoro privileges to the `core-ios-team` team.

Comment (jverkoey) [reply contributing/README.md]: I've updated that wording accordingly below. The mechanism by which CI automatically runs is gated by our teams though so I'll keep that wording above.

File: contributing/README.md
Comment (romoore) [reply contributing/README.md:37]: I don't think we should be documenting this as a feature/process. I think it's better to list it as an outstanding bug in the tooling.

Comment (jverkoey) [reply contributing/README.md:37]: Which aspect of this PR are you referring to?

Comment (romoore) [reply contributing/README.md:37]: This new addition is part of a "General Contributing Guidelines". If we consider kokoro's behavior to be a defect/bug, then this should probably be listed under and described as a caveat/faq/mitigation rather than as a guideline.

Comment (jverkoey) [reply contributing/README.md:37]: *nods* that's fair. At least for now though this is the behavior that we have to work within. I'll list the bug you filed here so that we can follow-up.

**Review by galiak11 — APPROVED** [review]

**Comment by jverkoey** [comment]
Waiting for LGTM from @randallli, @romoore, @yarneo, @andrewoverton, @wenyuzhang666.

**Review by yarneo — APPROVED** [review]

File: contributing/README.md
Comment (ianegordon) [inline contributing/README.md]: This seems like additional work for the team.

Can we not have CI run on all PRs?
Is the concern that external contributors won't be able to open detailed results?

**Review by ianegordon — DISMISSED** [review]

File: contributing/README.md
Comment (jverkoey) [reply contributing/README.md]: Unfortunately this is not something we'll be able to work around - see b/115506452 for additional details.

Comment (ianegordon) [reply contributing/README.md]: Got it, this is a requirement.

**Review by ianegordon — APPROVED** [review]

**Review by fluxminds — APPROVED** [review]

**Comment by jverkoey** [comment]
Waiting for  @andrewoverton @randallli

**Review by andrewoverton — APPROVED** [review]

**Comment by jverkoey** [comment]
Waiting for @randallli

**Review by romoore — APPROVED** [review]

**Review by randallli — APPROVED** [review]

**Pull request merged**
